static/js/stream_list.js

```diff
@@ -269,6 +274,7 @@ exports.create_sidebar_row = function (sub) {
         return;
     }
     build_stream_sidebar_row(sub);
+    $('.stream-name[data-toggle="tooltip"]').tooltip();
```

Comment:
something along the lines of `$('#stream_filters .stream-name[data-toggle="tooltip"]').tooltip();` ?